tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1199,6 +1207,10 @@ When(
   }
 )
 
+When('the user uses the move button to move files using the webUI', function() {
```

Comment:
```suggestion
When('the user selects move to move files using the webUI', function() {
```
How about this? :thinking:

Reply:
IMO it would be better if we have something like this `When('the user selects move button to move files using the webUI', function() {`. For me the `selects move to move` seems a bit confusing.